app/templates/components/forms/events/view/create-access-code.hbs

```diff
@@ -60,65 +40,41 @@
     {{/each}}
   </div>
   <div>
-    <div class="inline field">
-      <UiCheckbox
-        @class="toggle"
-        @label={{t "Show more options"}}
-        @checked={{this.showMoreOptions}}
-        @onChange={{action (mut this.showMoreOptions)}} />
-    </div>
-    {{#if this.showMoreOptions}}
-      {{t 'Set the min and max quantity allowed to purchase per order (Optional)'}}
-      <div class="{{unless this.device.isMobile 'fields'}}">
-        <div class="wide field {{if this.device.isMobile 'sixteen' 'five'}}">
-          <label>{{t 'Min'}}</label>
-          <Input
-            @type="number"
-            @name="min"
-            @min={{0}}
-            @value={{this.data.minQuantity}} />
-        </div>
-        {{#if this.device.isMobile}}
-          <div class="ui hidden divider"></div>
-        {{/if}}
-        <div class="wide field {{if this.device.isMobile 'sixteen' 'five'}}">
-          <label>{{t 'Max'}}</label>
-          <Input
-            @type="number"
-            @name="max"
-            @min={{0}}
-            @value={{this.data.maxQuantity}} />
-        </div>
-      </div>
-      <div class="ui hidden divider"></div>
-      {{t 'Validity Period (Optional)'}}
-      <div class="fields">
-        <div class="wide field {{if this.device.isMobile 'sixteen' 'five'}}">
-          <label>{{t 'Valid from'}}</label>
-          <Widgets::Forms::DatePicker
+      <div class="field">
+          <label>{{t 'Access Valid from'}}</label>
```

Comment:
Pls correct the indentation

Reply:
Sorry I can't understand please elaborate

Reply:
Indentation refers to the spaces at the beginning of a code line.

Reply:
o sorry I will fix this

Reply:
I'd  just did it